microsoft/testsuites/dpdk/dpdktestpmd.py

```diff
@@ -255,8 +257,21 @@ def get_installed_version(self) -> VersionInfo:
             "libdpdk", update_cached=True
         )
 
+    __devname_files = ["main.c", "Makefile", "meson.build"]
+
     def _install(self) -> None:
         super()._install()
+        # copy devname application into the examples directory
+        devname_local_folder = PurePath(__file__).parent.joinpath("devname")
+        self._node.shell.mkdir(
+            self.asset_path.joinpath("examples/devname"), parents=False, exist_ok=False
```

Comment:
I might not want to have exist_ok=False? But it would be a good check if I end up trying to build DPDK twice without wiping the directory... Not sure.